wasm/hal-wasm.go

```diff
@@ -31,14 +31,17 @@ func (render *WASMHAL) updateKeyStateFrom(keyMap *map[uint32]bool) {
 
 func (render *WASMHAL) BlitPage(buffer [anotherworld.WIDTH * anotherworld.HEIGHT]anotherworld.Color, posX, posY int) {
 	// logger.Debug(">VID: BLITPAGE %d %d", posX, posY)
+	render.canvas.SetColor(buffer[0])
+	render.lastSetColor = buffer[0]
+	offset := 0
```

Comment:
I found myself wondering, what offset? should we call it xYOffset or a similarly more descriptive variant

Reply:
there is only one offset in the whole buffer, we could rename it to xYOffset but imho this does not improve the readability - its just an offset